my_project/tab_data_explorer/app_data_explorer.py

```diff
@@ -34,6 +34,7 @@
 explore_dropdown_names.update(sun_cloud_tab_dropdown_names.copy())
 explore_dropdown_names.update(more_variables_dropdown.copy())
 explore_dropdown_names.update(sun_cloud_tab_explore_dropdown_names.copy())
+explore_dropdown_names.pop("None", None)
```

Comment:
great, could you please make sure that this is a deep copy of the dictionary otherwise this change may affect the whole application. I see that you used .copy() above but maybe a deep copy is even better

Reply:
Hi Federico, I use the deepcopy function of the copy module to create a deep copy of the dictionary.